app/components/operator-form/component.js

```diff
@@ -1,6 +1,24 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
+	// isIncluded: Ember.computed('model.include_in_changeset', function() {
+	// 	if (this.get('operator.include_in_changeset') === true) {
+	// 		return true;
+	// 	} else {
+	// 		return false;
+	// 	}
+	// }),
+	isIncluded: true,
```

Comment:
Is `isIncluded` necessary as a separate property? Why not just bind to `operator.include_in_changeset` in the template? (To do that might require setting a default value of true for the `include_in_changeset` attribute on the `Operator` model.)